src/js/jsx/sections/style/EffectsPanel.jsx

```diff
@@ -94,22 +94,24 @@ define(function (require, exports, module) {
             event.stopPropagation();
         },
 
-        shouldComponentUpdate: function (nextProps) {
-            if (this.props.disabled !== nextProps.disabled ||
-                this.props.active !== nextProps.active) {
-                return true;
-            }
-
-            if (!Immutable.is(this.props.document.layers.selected,
-                nextProps.document.layers.selected)) {
-                return true;
-            }
-
-            if (!nextProps.visible && !this.props.visible) {
+        shouldComponentUpdate: function (nextProps, nextState) {
+            // Only compare the existence of clipboard.
+            var clipboardChanged = !!this.state.clipboard !== !!nextState.clipboar,
```

Comment:
Also, this is incredibly minor, but in this sCU and in AppearancePanel, it looks like this variable isn't read until after another check which could return early. If so, we should delay defining the variable until just before it's needed.